kymatio/backend/torch_backend.py

```diff
@@ -3,6 +3,13 @@
 
 BACKEND_NAME = 'torch'
 
+def input_checks(x):
+    if x is None:
+        raise TypeError('The input should be not empty')
```

Comment:
Period.

Also, why not just make sure it's a `torch.Tensor`? Just checking for `None` is quite restricted.

Reply:
@edouardoyallon